tools/pylint-extensions/azure-pylint-guidelines-checker/tests/test_pylint_custom_plugins.py

```diff
@@ -4433,10 +4433,11 @@ def test_docstring_varag_no_type(self):
         # Error on documenting keyword only args as param after *args in docstring
         node = astroid.extract_node(
             """
-            def function_foo(*x, y):
```

Comment:
Do we have any tests with a client that has multiple methods that violate the rule?